<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleSourcePathsForm</class>
 <widget class="QWidget" name="SingleSourcePathsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кратчайшие пути до вершины</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="vertexChoiseLabel">
         <property name="text">
          <string>Выбор начальной вершины:</string>
         </property>
         <property name="buddy">
          <cstring>vertexChoiseBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="vertexChoiseBox">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pathSearchButton">
       <property name="text">
        <string>Начать поиск</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="algorythmNoteLabel">
     <property name="text">
      <string>Результат работы алгоритма Дейкстры не будет правильным, в случае наличия отрицательных ребер в графе.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="distanceLabel_2">
     <property name="text">
      <string>Вектор растояний:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="shortestDistanceVector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="shortestPathsLabel">
     <property name="text">
      <string>Вектор восстановления путей:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="pathRenewVector">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
